src/main/java/net/dv8tion/jda/api/Permission.java

```diff
@@ -60,6 +67,7 @@
     MESSAGE_EXT_EMOJI(       18, true, true, "Use External Emojis"),
     USE_APPLICATION_COMMANDS(31, true, true, "Use Application Commands"),
     MESSAGE_EXT_STICKER(     37, true, true, "Use External Stickers"),
+    VOICE_MESSAGE_SEND(      46, true, true, "Send Voice Messages"),
```

Comment:
This is likely gonna be confusing. People might interpret this as text-in-voice instead.

Reply:
"Send Audio Message" then? Since bots don't have voices 😄